R/frosting.R

```diff
@@ -27,10 +27,17 @@
 #' p1 <- predict(wf1, latest)
 #' p1
 #'
-#' # Remove frosting from the workflow and predict
-#' wf2 <- wf1 %>% remove_frosting()
+#' # Update frosting in a workflow and predict
```

Comment:
one of these examples should do something that results in different predictions.